pandas-stubs/core/frame.pyi

```diff
@@ -2311,7 +2314,7 @@ class DataFrame(NDFrame, OpsMixin, _GetItemHack):
         index: _bool = ...,
         na_rep: _str = ...,
         formatters: FormattersType | None = ...,
-        float_format: Callable[[float], str] | None = ...,
+        float_format: FloatFormatType | Callable[[float], str] | None = ...,
```

Comment:
`FloatFormatType` is too wide, as it includes any `Callable`.  

Can you change `FloatFormatType` to use `Callable[[float], str]` instead of `Callable` in `_typing.pyi`, and then you don't need the `Callable` declaration here or elsewhere?

Reply:
I was going to suggest that indeed since we should collapse those two parts.
This is addressed.